infra/modules/database/dms-instance.tf

```diff
@@ -24,10 +24,10 @@ resource "aws_dms_endpoint" "target_endpoint" {
   certificate_arn                 = "arn:aws:dms:us-east-1:315341936575:cert:GWOIQRTIVQVRBL5ERMCKTUPHMM33MMDGIP57J4I"
   database_name                   = "app"
   endpoint_id                     = "api-dev-primary"
-  endpoint_type                   = "source"
+  endpoint_type                   = "target"
   engine_name                     = "aurora-postgresql"
   kms_key_arn                     = aws_kms_key.dms_endpoints.arn
-  secrets_manager_access_role_arn = data.aws_iam_role.dms_access.arn
+  secrets_manager_access_role_arn = aws_iam_role.dms_access.arn
   ssl_mode                        = "verify-ca"
   secrets_manager_arn             = data.aws_secretsmanager_secret.target_db.arn
```

Comment:
I think this is the same secret as the one defined in `role-manager.tf` as `data "aws_secretsmanager_secret" "db_password" { ... }`. Should we re-use that one, then remove lines 52-55 below with the hard-coded name?

Reply:
Mhmm! I'm planning on doing that next o7

Reply:
https://github.com/HHS/simpler-grants-gov/pull/1129/commits/b3ec778ee46edd0d34db7884502ae53da43a7f99